legal-api/src/legal_api/services/business_details_version.py

```diff
@@ -462,6 +463,7 @@ def get_name_translations_revision(filing, legal_entity_id) -> dict:
         name_translations_version = history_cls(AlternateName)
         name_translations_history = (
             db.session.query(name_translations_version)
+            .options(defer(name_translations_version.changed))
```

Comment:
Hii Argus, for here, I already deferred the changed column, but still cannot pass `name_translations_list = name_translations_current.union(name_translations_history).all()`, stuck at here...

Reply:
and besides this part, I have lots of overlapped with Kevin's ticket, maybe I will see what Kevin did